.circleci/config.yml

```diff
@@ -3,110 +3,232 @@ version: 2.0
 defaults: &defaults
   working_directory: /go/src/github.com/sylabs/singularity-cri
   docker:
-    - image: sylabsio/golang:1.11-stretch
+    - image: golang:1.11-stretch
 
 jobs:
   get_source:
     <<: *defaults
     steps:
       - checkout
-      - restore_cache:
-          key: dep-cache-{{ checksum "Gopkg.lock" }}
-          paths:
-            - vendor
       - run:
           name: Install dep tool
-          command: |-
+          working_directory: /
+          command: |
+            mkdir -p $GOPATH/bin
             curl -s https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run:
-          name: Set up vendor directory
-          command: |-
+          name: Ensure vendored dependencies
+          command: |
```

Comment:
Suggest using `dep check` rather than `dep ensure` here. The former is more efficient because it only verifies that your imports, `vendor/`, `Gopkg.lock` and `Gopkg.toml` are in sync, whereas the latter tries to fix the problems.